readme.md

```diff
@@ -10,7 +10,7 @@ Lighthouse is integrated directly into the Chrome Developer Tools, under the "Au
 
 **Run it**: open Chrome DevTools, select the Audits panel, and hit "Run audits".
 
-<img width="500px" alt="Lighthouse integration in Chrome DevTools" src="https://user-images.githubusercontent.com/2301202/40556658-4ef7d128-6002-11e8-903e-5224894a7cca.png">
+<img src="https://raw.githubusercontent.com/GoogleChrome/lighthouse/HEAD/assets/example_dev_tools.png" alt="Lighthouse integration in Chrome DevTools" width="500px">
```

Comment:
I'm not 100% sure what `../HEAD/` will do some maybe you can help me understand :)

I'll restate what I'm worried about too. Scenario: 

1. We publish v3.x with `../HEAD/..` (or master) links.
2. We continue working and update the images or remove them entirely.
3. HEAD and master now point to a different image or is broken link.

I probably missed some magic described in that doc but seems like we still need a specific tag or hash in there for them to continue to work. Not as big a deal as doc links IMO but should probably iron out if we're switching them all anyhoo :)

Reply:
HEAD is basically an alias to the default branch, which is master for us.  It's just a way of taking out master from the url IIRC, but I can't find any supporting docs :frowning: 

That is a valid concern.  If you look at the readme on this branch you can tell it is broken b/c `example_dev_tools.png` doesn't exist on master yet.

Looking into what github has for [permanent links](https://help.github.com/articles/getting-permanent-links-to-files/) we could do that e.g. `https://raw.githubusercontent.com/GoogleChrome/lighthouse/e7997b3db01de3553d8cb208a40f3d4fd350195c/assets/example_dev_tools.png` link to the file as seen on a specific commit, thus guaranteeing it never breaks going forward.  But it opens it up to getting stale and the update procedure is more complicated than just sticking in a new image to the assets folder.

I think that if we do change it to this, or even keep it the same, we need to document this process b/c for someone like me with the user-content hash'd URLs I had no idea how to update them.  So I think moving to something like this, where the images exist and have discernible URLs would be helpful IMO.